lib/config/wrapper.go

```diff
@@ -142,7 +142,7 @@ func (w *Wrapper) RawCopy() Configuration {
 func (w *Wrapper) Replace(cfg Configuration) (Waiter, error) {
 	w.mut.Lock()
 	defer w.mut.Unlock()
-	return w.replaceLocked(cfg)
+	return w.replaceLocked(cfg.Copy())
```

Comment:
I think this copy and the one in `replaceLocked` are redundant.

Reply:
Not really, as `to` is not copied when stored on `w.cfg`, so the called still can fiddle with original slices passed in `wrapper.Replace(cfg)`